policyengine_us/parameters/gov/local/ca/la/dss/expectant_parent_payment/amount.yaml

```
@@ -0,0 +1,12 @@
+description: The California Department of Social Services provides the following Expectant Parent Payment amount, based on pregnancy months. 
```

`based on` usually for breakdowns
```suggestion
description: The California Department of Social Services provides the following Expectant Parent Payment. 
```